postgresql-persistence/src/main/java/com/netflix/conductor/aurora/log4j/DeluxeAuroraAppender.java

```diff
@@ -115,7 +112,17 @@ public void init() {
 		}
 	}
 
+	private void shutdownListener() {
+		Signal.handle(new Signal("TERM"), sig -> {
```

Comment:
Is there any other Signal we need to worry about?
Also missing call to this.close()